pandapower/pypower/idx_ssc.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 1996-2015 PSERC. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
```

Comment:
Is the copyright, similar to other pypower copyrights, needed for SSC indices, too?

Reply:
True, should be removed